implementation/src/main/java/io/smallrye/jwt/auth/principal/JWTCallerPrincipal.java

```diff
@@ -192,6 +194,8 @@ protected JsonValue wrapClaimValue(Object value) {
             for (Object element : list) {
                 if (element instanceof String) {
                     arrayBuilder.add(element.toString());
+                } else if (element == null) {
+                    arrayBuilder.add(NULL);
```

Comment:
@nderwin thanks, this looks good, only a minor cosmetic issue: NULL is statically imported while just few lines above JsonValue.TRUE/FALSE are used. Lets make it consistent, I don't mind much if the static imports will be used, though have a minor preference for the `JsonValue` qualification in the code as it is easier to see what kind of NULL or Boolean values the code is dealing with, but you choose :-)

Reply:
Ah, IDE auto-imports strike again.